doc/Installation/Installation-Ubuntu-1604-Nginx.md

```diff
@@ -36,9 +38,17 @@ lower_case_table_names=0
 
 #### Install / Configure Nginx
 
-`apt-get install php7.0-cli php7.0-mysql php7.0-gd php7.0-snmp php-pear php7.0-curl php7.0-fpm snmp graphviz php7.0-mcrypt php7.0-json nginx-full fping imagemagick whois mtr-tiny nmap python-mysqldb snmpd php-net-ipv4 php-net-ipv6 rrdtool git`
+`sudo apt-get install php7.0-cli php7.0-mysql php7.0-gd php7.0-snmp php-pear php7.0-curl php7.0-fpm snmp graphviz php7.0-mcrypt php7.0-json nginx-full fping imagemagick whois mtr-tiny nmap python-mysqldb snmpd php-net-ipv4 php-net-ipv6 rrdtool git curl`
+
+Add the timezone to the following files - ensure `date.timezone` is set to your preferred time zone.  See http://php.net/manual/en/timezones.php for a list of supported timezones.  
+
+Valid examples are: `date.timezone = America/New_York`, `date.timezone = Australia/Brisbane`, `date.timezone = Etc/UTC`.
+
+`nano /etc/php/7.0/fpm/php.ini`
+
+`nano /etc/php/7.0/cli/php.ini`
 
-In `/etc/php/7.0/fpm/php.ini` and `/etc/php/7.0/cli/php.ini`, ensure date.timezone is set to your preferred time zone.  See http://php.net/manual/en/timezones.php for a list of supported timezones.  Valid examples are: "America/New_York", "Australia/Brisbane", "Etc/UTC".
+restart php to roll in changes
```

Comment:
You don't need to tell people to do this, the command provided below is there for that.